### PR TITLE
Adjust wait_for_cooling slope

### PR DESCRIPTION
**Pull request by AnHardt**
Adjust wait_for_cooling slope
and drop mintemp for cooling.

See
https://github.com/MarlinFirmware/Marlin/pull/4169#issuecomment-231544532

### CODE REVIEW

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -4562,6 +4562,13 @@ inline void gcode_M105() {
 
 #endif
 
+  #if DISABLED(MIN_COOLING_SLOPE_DEG)
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp]: We have to use `#ifndef` for settings that take values.

**Comment by thinkyhead** [comment]
Overall it looks good. Do we need to increase one of our vars here to `long` as you mentioned?

File: Marlin/Marlin_main.cpp [hunk 2]
```diff
@@ -4769,11 +4783,11 @@ inline void gcode_M109() {
...
       if (wants_to_cool) {
-        if (temp < 30.0) break; // always break at 30°
-        // break after 20 seconds if cooling stalls
+        // break after MIN_COOLING_SLOPE_TIME_BED seconds
+        // if the temperature did not drop at least MIN_COOLING_SLOPE_DEG_BED
         if (!next_cool_check_ms || ELAPSED(now, next_cool_check_ms)) {
-          if (old_temp - temp < 1.0) break;
-          next_cool_check_ms = now + 20000;
+          if (old_temp - temp < MIN_COOLING_SLOPE_DEG_BED) break;
+          next_cool_check_ms = now + 1000UL * MIN_COOLING_SLOPE_TIME_BED;
```
Comment (AnHardt) [inline Marlin/Marlin_main.cpp:4730]: @thinkyhead 
No, not a variable. Just a constant. For some reason `1000UL * MIN_COOLING_SLOPE_TIME_BED` is done in int if we don't  us UL. Worked great for times up to 32 seconds, but not for 60.

File: Marlin/Marlin_main.cpp (hunk 1 above)
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp]: We have to use `#ifndef` for settings that take values. So, once it's patched this up I'll merge this asap!

**Pull request merged**